README.md

```diff
@@ -1,10 +1,11 @@
 # aws_ssm_provider
 
-A configuration provider for Distillery which handles AWS SSM parameters.
+A configuration provider for Mix which handles AWS SSM parameters.
 
 ## Usage
 
-This package is intended to be used in conjunction with [Distillery Config Providers](https://hexdocs.pm/distillery/config/runtime.html#config-providers).
+This package is intended to be used in conjunction with Mix Config.Providers in Elixir 1.9+
```

Comment:
Question: do we want to note that in `v1.0`, we supported Distillery, but as of `v2.0` we only support the native Mix releases?

Reply:
@dillonius01 That's what line 76 in the README is for.